azure-pipelines.yml

```diff
@@ -0,0 +1,10 @@
+trigger:
+- wrf-cmake
```

Comment:
Can we change this to run on all branches as well?

Reply:
Done.